tasks/sign-binaries/README.md

```diff
@@ -10,4 +10,11 @@ Tekton task to sign windows and mac binaries before they are pushed to the Red H
 | quaySecret | Secret to interact with Quay | No |  |
 | windowsCredentials | Secret to interact with the Windows signing host | No |  |
 | windowsSSHKey | Secret containing private key and fingerprint for Windows signing host | Yes | windows-ssh-key |
+| macHostCredentials | Secret to interact with the Mac signing host | No |  |
+| macSigningCredentials | Secret to interact with the Mac signing utils | No |  |
+| macSSHKey | Secret containing SSH private key for the Mac signing host | Yes | mac-ssh-key
```

Comment:
Nit pick but missing `|` at the end of this line

Reply:
fixed, thanks!